File: tracks/python/resistor_color_duo.py
color_codes = {"black": 0, "brown": 1, "red": 2, "orange": 3, "yellow": 4,
               "green": 5, "blue": 6, "violet": 7, "grey": 8, "white": 9}

def value(colors):
    codes = ""
    for color in colors[:2]:
        codes += str(color_codes.get(color))
    return int(codes)

print(value(["brown", "green", "violet"]))

File: tracks/typescript/resistor-color-duo.ts
const colorCode = (color:string):Number => {
    return COLORS.indexOf(color);
}

const COLORS = ['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white',];

export function decodedValue(colors: string[]) {
    return Number(colorCode(colors[0]) + "" + colorCode(colors[1]))
}